lib/ansible/modules/cloud/docker/docker_compose.py

```diff
@@ -717,6 +717,29 @@ def cmd_up(self):
             result['changed'] = build_output['changed']
             result['actions'] += build_output['actions']
 
+        if self.remove_orphans:
+            # containers = list(filter(None, [
+            #     Container.from_ps(self.project.client, container)
+            #     for container in self.project.client.containers(
+            #         all=False,
+            #         filters={'label': ['{0}={1}'.format(LABEL_PROJECT, self.project.name), '{0}={1}'.format(LABEL_ONE_OFF, "False")]})])
+            # )
+
+            containers = self.client.containers(
+                filters={'label': ['{0}={1}'.format(LABEL_PROJECT, self.project.name), '{0}={1}'.format(LABEL_ONE_OFF, "False")]}
+            )
+
+            def _findOrphans():
+                for container in containers:
+                    # service_name = ctnr.labels.get(LABEL_SERVICE)
+                    service_name = container['Labels'][LABEL_SERVICE]
```

Comment:
```suggestion
                    service_name = container.get('Labels', {}).get(LABEL_SERVICE)
```
So that the module won't crash'n'burn when `Labels` isn't present (though that shouldn't happen since you filtered by labels) or `LABEL_SERVICE` is not a label of the container.